linebot/api.py

```diff
@@ -327,6 +330,176 @@ def get_progress_status_narrowcast(self, request_id, timeout=None):
 
         return MessageProgressNarrowcastResponse.new_from_json_dict(response.json)
 
+    def validate_reply_message_objects(self, messages, timeout=None):
+        """Call validate reply message objects API.
+
+        https://developers.line.biz/en/reference/messaging-api/#validate-message-objects-of-reply-message
+
+        You can validate that an array of message objects is valid as a value
+        for the messages property of the request body for the send reply message endpoint.
+
+        :param messages: Messages.
+            Max: 5
+        :type messages: T <= :py:class:`linebot.models.send_messages.SendMessage` |
+            list[T <= :py:class:`linebot.models.send_messages.SendMessage`]
+        :param timeout: (optional) How long to wait for the server
+            to send data before giving up, as a float,
+            or a (connect timeout, read timeout) float tuple.
+            Default is self.http_client.timeout
+        :type timeout: float | tuple(float, float)
+        :rtype: :py:class:`linebot.models.responses.ValidateReplyMessageObjectsResponse`
+        """
+        if not isinstance(messages, (list, tuple)):
+            messages = [messages]
```

Comment:
If the given instance is invalid, the line fails or `POST /v2/bot/message/validate/reply` returns error response finally.

I just keep current `reply_message` implementation because I guess sdk user passes same message objects to `reply_message` and `validate_reply_message_objects`. If `validate_reply_message_objects` has different implementation, I think sdk user can be confused. It seems more clear that both methods have the same implementation, though the current implementation itself seems confusing :sweat:.
What do you think about it?

(`reply_message`)
https://github.com/line/line-bot-sdk-python/blob/6073cade55ec31b4ba1ab405b9ad894165f8fbce/linebot/api.py#L76-L105